docs/pages/includes/install-windows.mdx

```diff
@@ -5,20 +5,19 @@ To install `tsh` and `tctl` on Windows, run the following commands in
 **PowerShell** (these commands will not work in `cmd.exe`):
 
 <Tabs>
-<TabItem label="Teleport Community Edition" scope="oss">
+<TabItem label="Teleport Enterprise Cloud" scope="cloud">
```

Comment:
Since the content of each tab item is so similar in this `Tabs` component, I think it would make sense to collapse all tab items into one, then add explicit instructions to choose a binary/version based on your edition.